bazarr/sonarr/sync/series.py

```diff
@@ -16,6 +16,20 @@
 from .parser import seriesParser
 from .utils import get_profile_list, get_tags, get_series_from_sonarr_api
 
+# map between booleans and strings in DB
+bool_map = {"True": True, "False": False}
+
+FEATURE_PREFIX = "SYNC_SERIES "
+def trace(message):
+    if settings.general.debug:
+        logging.debug(FEATURE_PREFIX + message)
```

Comment:
Any chance to not duplicate that part of code and have it once somewhere like bazarr/utilities/helper.py?

Reply:
I would have loved to do so, but I didn’t know where to put it.
I’ll put it there if that’s the best place.

Can I also create a constant for the magic number 20480?
That one has been bothering me.

Reply:
Forget code duplication, there's no real benefit in this specific case as you need to deal with context.

Yes adding a constant for 20480 in constants.py would be a good idea.

Reply:
Do you prefer lower case or all caps for constants?
e.g.

\# minimum file size for Bazarr to consider it a video
MINIMUM_VIDEO_SIZE = 20480

Reply:
I've never did it that way but it seems to be the correct way of doing it. Lets go with uppercase and we'll fix the others in time.

Reply:
Done.